plenum/server/replica.py

```diff
@@ -2760,3 +2760,10 @@ def _get_last_timestamp_from_state(self, ledger_id):
                                       "".format(last_timestamp))
                     return last_timestamp
         return None
+
+    def get_ledgers_last_update_time(self)->dict:
+        if self._freshness_checker:
+            return self._freshness_checker.get_last_update_time()
+
+    def get_time_for_3pc_batch(self)->int:
```

Comment:
This is a matter of taste, but I prefer to have this as a field in constructor that can be overridden. This makes the tests more clean from my opinion.